Lombiq.Tests.UI/Extensions/VisualVerificationUITestContextExtensions.cs

```diff
@@ -0,0 +1,509 @@
+using Codeuctivity.ImageSharpCompare;
+using Lombiq.Tests.UI.Attributes;
+using Lombiq.Tests.UI.Exceptions;
+using Lombiq.Tests.UI.Models;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+using Shouldly;
+using SixLabors.ImageSharp.Processing;
+using System;
+using System.Diagnostics;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Lombiq.Tests.UI.Extensions;
+
+public static class VisualVerificationUITestContextExtensions
+{
+    private const string ConditionLessThenOrEqual = "less then or equal to";
+
+    /// <summary>
+    /// Compares the reference image and screenshot of the element given by <paramref name="elementSelector"/>. The mean
+    /// error percentage should be less then or equal with the given <paramref name="meanErrorPercentageThreshold"/>.
+    /// The reference image is automatically loaded from assembly resource, if it's not exists then from the project
+    /// path based on <see cref="VisualMatchApprovedConfiguration"/> - it can be configured over
+    /// <paramref name="configurator"/> -, if the reference image doesn't exist, a new one will be created based on the
+    /// element's screenshot, and an <see cref="VisualVerificationReferenceImageCreatedException"/> will be thrown. The
+    /// reference image path is generated from the method name - annotated with
+    /// <see cref="VisualVerificationAttribute"/> - and the source file name and path, where the method is.
```

Comment:
Do the same fixes for the other overloads too.

```suggestion
    /// Compares the reference image and screenshot of the element given by <paramref name="elementSelector"/>. The mean
    /// error percentage should be less than or equal to the given <paramref name="meanErrorPercentageThreshold"/>.
    /// The reference image is automatically loaded from assembly resource, if it doesn't exist then from the project
    /// path based on <see cref="VisualMatchApprovedConfiguration"/> - it can be configured over
    /// <paramref name="configurator"/> -, if the reference image doesn't exist, a new one will be created based on the
    /// element's screenshot, and an <see cref="VisualVerificationReferenceImageCreatedException"/> will be thrown. The
    /// reference image path is generated from the method name - annotated with
    /// <see cref="VisualVerificationAttribute"/> - and the source file name and path, where the method is.
```